### DISCUSSION TITLE
Could not save pattern to temporary directory

### DISCUSSION THREAD

**Discussion by Djard** [hydrogen-music/hydrogen#1484] [Hydrogen on Windows]
I get the failure notice when attempting to duplicate or delete a pattern track. I notice that others have posted the issue as far back as 2020. So I figure a bug exists. Working on permissions was reported as the cause, but nobody in the thread found a solution. Anything else I can try?

Hydrogen v.1.1.1 on Win 10 Pro x64

**Comment by cme** [comment]
Where is this thread?

This sounds like #1425 which has been fixed on master. As a workaround, make sure pattern names are valid Windows filenames (Windows has a fairly restrictive set of allowable characters in filenames, unfortunately)

**Reply by Djard** [reply]
I closed the project and started afresh. Now the issue does not appear. If you Google search for "Could not save pattern to temporary directory," you will see others discussing the matter. One such post is on the following page:

https://github.com/hydrogen-music/hydrogen/issues/941
